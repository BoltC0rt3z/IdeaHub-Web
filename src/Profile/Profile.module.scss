@import '../styles/styles.module.scss';

.profileContainer {
  background: $color-light;

  .profileImage {
    padding-left: 16px;
    padding-right: 16px;

    .imageContainer {
      margin-top: -120px;

      .image {
        width: 136px;
        height: 136px;
        object-fit: cover;
        border-radius: 50%;
        border: 5px solid $color-light;
      }

      @include respond-to('xs-l') {
        margin-top: -80px;

        .image {
          width: 80px;
          height: 80px;
        }
      }
    }

    .editProfileButton {
      margin-top: -60px;
    }
  }

  .coverImageContainer {
    height: 200px;

    .coverImage {
      width: 100%;
      height: 100%;
      object-fit: cover;
    }
  }
}

.nameContainer {
  color: $color-dark-blue-lighter;
  padding-left: 28px;
  padding-bottom: 12px;

  .name {
    font-weight: bold;
    font-size: 20px;
  }
}

.iconsContainer {
  padding-right: 16px;
  padding-top: 16px;
}

.message {
  padding: 12px 0px 0px 13px;
  color: $color-dark-blue-lighter;
  width: 48px;
  height: 48px;
  border-radius: 50%;
  border: 1px solid $color-dark-blue-lighter;
  margin-top: -8px;
  cursor: pointer;
  transition: background 20ms;

  &:hover {
    color: $color-light;
    background: $color-dark-blue-lighter;
  }
}
