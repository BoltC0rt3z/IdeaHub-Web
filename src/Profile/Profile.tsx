import React from 'react';

const Profile: React.FC = () => {
  return <div>ds</div>;
};

export default Profile;
