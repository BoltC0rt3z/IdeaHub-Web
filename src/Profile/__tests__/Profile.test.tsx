import React from 'react';
import { mount, ReactWrapper } from 'enzyme';

import Profile from '../Profile';

describe('Tests for profile', () => {
  const wrapper: ReactWrapper = mount(<Profile />);

  it('Should render profile component without crushing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
