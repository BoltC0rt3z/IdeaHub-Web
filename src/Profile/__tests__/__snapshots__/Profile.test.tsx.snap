// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for profile Should render profile component without crushing 1`] = `ReactWrapper {}`;
